packages/platform-ios/src/link/getTargets.ts

```diff
@@ -4,20 +4,19 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @format
  */
 
 /**
  * Given xcodeproj it returns list of targets
  */
-export default function getTargets(project) {
+export default function getTargets(project: any) {
   const {
     firstProject: {targets},
   } = project.getFirstProject();
   const nativeTargetSection = project.pbxNativeTargetSection();
   return targets
-    .filter(target => nativeTargetSection[target.value] !== undefined)
-    .map(target => {
+    .filter((target: any) => nativeTargetSection[target.value] !== undefined)
```

Comment:
Would something like this work?
```suggestion
    .filter((target: { value: string; }) => nativeTargetSection[target.value] !== undefined)
```